great-idea/services.html

```diff
@@ -15,93 +15,112 @@
 </head>
 
 <body>
+  <header>
+    <div class="top">
+      <nav>
+          <a href="#">Services</a>
+          <a href="#">Product</a>
+          <a href="#">Vision</a>
+          <a href="#">Features</a>
+          <a href="#">About</a>
+          <a href="#">Contact</a>
+      </nav>
+      <img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
```

Comment:
You can also surround your img by an anchor tag (<a href="">) and link the image to an HTML or URL. In this instance, you can direct any click on the image to the home page by adding index.html.